### PR TITLE
Data Pull

### PR DESCRIPTION
**Pull request by AdmiringWorm**

### CODE REVIEW

File: Data/Data/windows.xml [hunk 1]
```diff
@@ -1050,6 +1071,20 @@
 		<save path="Profiles"/>
 		<contributor>slake_jones</contributor>
 	</game>
+  <game name="CSIHardEvidence">
```
Comment (sanmadjack) [inline Data/Data/windows.xml]: why isn't this marked as being windows platform?

File: Data/Data/windows.xml [hunk 2]
```diff
@@ -1050,6 +1071,20 @@
...
+    <title>CSI: Hard Evidence</title>
+    <location_path environment_variable="steamcommon" path="csi hard evidence"/>
+    <location_registry root="local_machine" key="HKEY_LOCAL_MACHINE\SOFTWARE\Telltale Games\CSI4.exe" value="Install Location"/>
+    <save filename="CSISaveGame*"/>
+    <save filename="prefs.prop" type="Settings"/>
+    <contributor>AdmiringWorm</contributor>
+  </game>
+  <game name="CSINYTheGame">
```
Comment (sanmadjack) [inline Data/Data/windows.xml]: windows platform

File: Data/Data/windows.xml (hunk 1 above)
Comment (AdmiringWorm) [inline Data/Data/windows.xml]: It was an oversight from my end. I'll update it in the next push.

**Comment by sanmadjack** [comment]
Okay, so update those two lines and I'll accept the pull :)

**Comment by AdmiringWorm** [comment]
ok, I'll do that. the next push will be some new additions as well as some updates. Just close the pull request and I'll open a new one when I've pushed the new data

**Pull request closed without merging**